src/app/App.js

```diff
@@ -47,4 +54,29 @@ export const App = ({ options }) => {
     modeHall.hidden = !modeHall.hidden;
     modeRules.hidden = !modeRules.hidden;
   }
+
+  // dynamicaly set active game mode
+  gameModes.forEach((mode) => {
+    mode.addEventListener('click', (e) => {
+      gameModes.forEach((mode) => {
+        mode.classList.remove('menu__item--selected');
+      });
+      e.target.classList.add('menu__item--selected');
+      selectedGame = document
+        .querySelector('.menu__item--selected')
+        .innerHTML.toLowerCase();
+    });
+  });
+  //get data from API based on active game mode
+  const quizData = new Quiz();
+  playTheGame.addEventListener('click', () => play(selectedGame));
+
+  function play(query) {
+    fetch(`https://swapi.dev/api/${query}/`)
+      .then((response) => response.json())
```

Comment:
nie lepiej żebyś pobrał samo name z obiektu?

Reply:
no i id ofc

Reply:
Te obiekty posluza juz do gry  , ale mozna ograniczyc pobieranie do {id, name, imgUrl}. Te dane powinny wystarczyc do calej gry.

Reply:
no o to mi chodzi, bez sensu pobierac cale

Reply:
Niemniej jednak bym zostawil, bo tam jest sporo ciekawych danych ktore mozemy pozniej wykorzystywac do rozbudowania gry. Ponadto jest tylko 10 obiektow na tryb, i ich waga jest znikoma.

Reply:
jest taka zasada programowania YAGNI - You Aren't Gonna Need It. jeśli w tym momencie tego nie potrzebujesz to tego nie wrzucaj. zawsze możesz dorzucić jak będzie potrzebne

Reply:
10 obiektów jest na jednej stronie, w elemencie next jest link do kolejnej podstrony wyników z kolejną dyszką. w elemencie count jest informacja, że łącznie są 82 obiekty w people

Reply:
Moglabys wskazac gdzie znajduje sie ten element ?

Reply:
Jak jesteś w trybie np. people to jest pole next i tam jest link do kolejnej stony. Niestety biały na jasnym tle, więc słabo widać

Reply:
Wszystkie dane na temat danej postaci już zostały pobrane, więc czy zatrzymamy je tutaj czy popchniemy w dół to ma niewielkie znaczenie (wiadomo, lepiej by było pobierać tylko to co potrzebujemy ale do tego było by potrzebne lepiej zrobione API np.: [link](https://graphql.org/swapi-graphql?query=%7B%0A%20%20allPeople%7B%0A%20%20%20%20people%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D) ale to już jest inna technologia więc na razie bym się nie szarpał). 

Ja bym się bardziej zastanowił nad logiką generowania kolejnych pytań w quizie. Tak jak już @lilareliga zauważyła to API pod ścieżką https://swapi.dev/api/people/ zwraca 10 elementów i informacje jak zdobyć resztę (tzw. [paginacja](https://nordicapis.com/everything-you-need-to-know-about-api-pagination/)). Obecnie mamy trochę nudny quiz bo zawsze mamy 10 pierwszych postaci w API. Możemy oczywiście naprawić to po najniższej linii oporu i wysłać 9 requestów do API podmieniając parametr page zapytania ale wtedy to się troszkę mija z celem bo zanim nasz quiz wystartuje i tak będzie musiał pobrać całą baze danych (wydajność 👎 ).

W sukurs przychodzą tutaj wymagania biznesowe aplikacji:

> Pytania są generowane w następujący sposób:
zostaje pobrany losowy zasób z danego trybu (np people o id 5)
zostanie pobrane dla wylosowanego zasobu zdjęcie
losowane są 3 odpowiedzi z zapytania do StarWars API. Dla trybu "People" będzie to: https://swapi.co/api/people (jedna brana jest z wcześniej wylosowanego, musi być poprawna)

Czyli endpoint który należy wykorzystać to: https://swapi.dev/api/people/1/ gdzie jeden to losowo wybrana liczba w zakresie od 1 do [liczebność zbioru dla danego zasobu] (tą wartość proponuje na razie zahardkodować potem można to pobierać osobnym requestem) z tego requesta mamy imie postaci o która chcemy zapytać w quizie. Następnie wybieramy trzy błędne imiona na jakieś losowej stronie (żeby nie dawać zawsze tych samych błędnych odpowiedzi).Tutaj trzeba tez przypilnować czy losowo wybrana błędna odpowiedź nie jest identyczna jak ta poprawna.

Logike odnosnie wybierania błędnych odpowiedzi można by było też troszkę inaczej zaimplementować niż jest to w wymaganiach ale to już możemy przegadać bo mi się trochę esej z tego komentarza zrobił.

Reply:
dobry pomysł! chcieliśmy wysłać na początek 9 requestów i zapisać dane w jakimś obiekcie, no i później z niego losować. to o czym piszesz jest prostsze i lepsze :P
na razie dorzuciłam małego fixa, zaktualizowałam do developa i robię merga, żeby kolejne osoby miały się od czego odbić